scripts/common.ts

```diff
@@ -70,23 +70,32 @@ export async function importDataModule(filePath: string): Promise<any[]> {
 
 export function runTestCommand(command: string): void {
   console.log(`Running command: ${command}`);
-  const testCommand = exec(command);
-  if (!testCommand.stdout || !testCommand.stderr) {
-    console.error('Failed to execute command: stdout or stderr is null');
-    return;
-  }
+
+  // Split command into executable and arguments to avoid shell interpretation
+  const parts = command.trim().split(/\s+/);
+  const executable = parts[0];
+  const args = parts.slice(1);
+
+  const testCommand = spawn(executable, args, {
+    stdio: ['pipe', 'pipe', 'pipe'],
+    shell: false, // Disable shell to prevent command injection
+  });
 
   testCommand.stdout.on('data', (data) => {
-    console.log(data);
+    console.log(data.toString());
```

Comment:
data will be json?

Reply:
The `data` is a Buffer containing stdout from the spawned process. The output format depends on the command being run - test runners typically output plain text, while API calls might return JSON.

Since this is a generic test command runner, I've kept `data.toString()` to handle any output format flexibly. If specific commands are expected to return JSON, we can add conditional parsing later.